scripts/variantstore/wdl/GvsBulkIngestGenomes.wdl

```diff
@@ -10,10 +10,11 @@ workflow GvsBulkIngestGenomes {
     input {
```

Comment:
I would take out all the comments for each input and make sure that the information within them is in the `parameter_meta` section; no need to have information about the inputs in two places.